docs/index.rst

```diff
@@ -479,6 +479,18 @@ features <https://docs.confluent.io/current/security.html>`__:
 
 There is an OpenLDAP server running in the demo, and each Kafka broker in the demo is configured with MDS and can talk to LDAP so that it can authenticate clients and Confluent Platform services and clients.
 
+Zookeeper has two listener ports:
+
++---------------+----------------+--------------------------------------------------------------------+--------+--------+
+| Name          | Protocol       | In this demo, used for ...                                         | zookeeper       |
++===============+================+====================================================================+=================+
+| N/A           | SASL/DIGEST-MD5| Validating trial license for |crest| and |sr|. (no TLS support)    | 2181            |
++---------------+----------------+--------------------------------------------------------------------+-----------------+
+| N/A           | SASL_PLAINTEXT | Broker communication (kafka1, kafka2)                              | 2182            |
```

Comment:
@sdanduConf is `SASL_PLAINTEXT` the correct protocol here?  Should it read instead `TLS`?

Reply:
> is SASL_PLAINTEXT the correct protocol here? Should it read instead TLS?

 Probably `mTLS` is my guess.

Reply:
updated to mTLS, thanks!